.nvmrc

```diff
@@ -0,0 +1 @@
+lts/fermium
```

Comment:
:question: Does the storybook strictly requires Node 14? It's in maintenance mode. The latest active LTS is 16, and the current is 18 https://nodejs.org/en/about/releases/

Reply:
Updated to Gallium (v16)